docs/getting_started/modin_vs_dask_vs_koalas.rst

```diff
@@ -0,0 +1,91 @@
+* indexing & ordering semantics, unlike DaskDF and Koalas that deviate from these semantics
+* eager execution, unlike DaskDF and Koalas that provide lazy execution
+
+As a result, Modin's coverage is `more than 90% <https://github.com/modin-project/modin#pandas-api-coverage>`_ of the pandas API, while DaskDF and Koalas' coverage is about 55%. 
+
+.. figure:: ../img/api_coverage_comparison.svg
+   :align: center
+   :alt: Percentage coverage of the pandas API after deduplication
+
+For more technical details please see our VLDB 2022 paper, referenced `here <https://people.eecs.berkeley.edu/~totemtang/paper/Modin.pdf>`_. 
+
+Brief Overview of DaskDF and Koalas
+-----------------------------------
+
+Dask's `DataFrame <https://docs.dask.org/en/stable/dataframe.html>`_ (DaskDF) is effectively a meta-frame, partitioning and scheduling many smaller ``pandas.DataFrame`` objects. Users construct a task graph of dataframe computation step by step and then trigger computation using the ``compute`` function.
+
+
+Spark's `Koalas <https://koalas.readthedocs.io/en/latest/>`_ provides the pandas API on Spark, leveraging the preexisting Spark SQL optimizer to execute select pandas commands. Like DaskDF, Koalas also employs lazy computation, only triggering computation when the user requests to see the results.
+
+
+
+Partitioning and Parallelization
+--------------------------------
+**DaskDF and Koalas only support row-oriented partitioning and parallelism.** This approach is analogous to relational databases. The dataframe is conceptually broken down into horizontal partitions along rows, where each partition is independently processed if possible. When DaskDF or Koalas are required to perform column-parallel operations that to be done on columns independently (e.g., dropping columns with null values via ``dropna`` on the column ``axis``), they either perform very poorly with no parallelism or do not support that operation.
+
+
+**Modin supports both row, column, and block-oriented partitioning and parallelism**. That is, the dataframe can be conceptually broken down as groups of rows, groups of columns, or both groups of rows and groups of columns (effectively a block or sub-matrix). Modin will transparently reshape the partitioning as necessary for the corresponding operation, based on whether the operation is row-parallel, column-parallel, or cell-parallel (independently applied to each unit cell). This allows Modin to support more of the pandas API and do so efficiently. Due to the finer-grained control over the partitioning, Modin can support a number of operations that are very challenging to parallelize in row-oriented systems (e.g., ``transpose``, ``median``, ``quantile``). This flexibility in partitioning also gives Modin tremendous power to implement efficient straggler mitigation and improve utilization over the entire cluster.
+
+
+API Coverage
+------------
+
```

Comment:
Possibly adding a header sentence saying something like *"One of the great things about pandas is the wide array of convenient functionalities, with more than 600+ APIs for XYZ. However, it is not trivial to implement all of these APIs in a scalable dataframe system."*

Reply:
Yes, rewording/expanding: "_One of the key benefits of pandas is its versatility, due to the wide array of operations, with more than 600+ API operations for data cleaning, feature engineering, data transformation, data summarization, data exploration, and machine learning. However, it is not trivial to develop scalable implementations of each of these operations in a dataframe system._"

Reply:
Looks great! Adding this in: 

```suggestion

One of the key benefits of pandas is its versatility, due to the wide array of operations, with more than 600+ API operations for data cleaning, feature engineering, data transformation, data summarization, data exploration, and machine learning. However, it is not trivial to develop scalable implementations of each of these operations in a dataframe system.
```